### PR TITLE
worker: use external worker package

### PR DESCRIPTION
**Pull request by rogpeppe**
The package gopkg.in/worker.v1 has existed for a while
and will soon have some changes that we need in Juju,
so use it instead of the locally defined Runner type
and delete that.

Also sort the imports canonically throughout.
Although large, there are no significant semantic
changes introduced by this PR.

### CODE REVIEW

File: state/workers/fixture_test.go [hunk 1]
```diff
@@ -11,10 +11,10 @@ import (
 	"github.com/juju/testing"
 	jc "github.com/juju/testing/checkers"
 	gc "gopkg.in/check.v1"
+	worker "gopkg.in/juju/worker.v1"
```
Comment (mjs) [inline state/workers/fixture_test.go]: Why the alias? (similarly below)

Comment (rogpeppe) [inline state/workers/fixture_test.go]: That's what goimports does by default, and who am I to argue?

**Comment by rogpeppe** [comment]
$$merge$$

**Comment by rogpeppe** [comment]
$$merge$$

**Pull request merged**